content-src/components/FluentOrText/FluentOrText.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+/**
+ * Set text on a child element/component depending on if the message is already
+ * translated plain text or a fluent id with optional args.
+ */
+export function FluentOrText({ children, message }) {
+  // Ensure we have a single child to attach attributes
+  const child = children ? React.Children.only(children) : <span />;
+
+  // For a string message, just use it as the child's text
+  let grandChildren = message;
+  let extraProps;
+
+  // Convert a message object to set desired fluent-dom attributes
+  if (typeof message === "object") {
+    extraProps = {
+      "data-l10n-id": message.id,
+      "data-l10n-args": message.values && JSON.stringify(message.values),
```

Comment:
I suppose we could make the `message` object provide `args` instead of `values` but somehow our existing messages use the `values` property that we would need to fix up all instances:
https://github.com/mozilla/activity-stream/blob/db21a369dd5cccff3c9cfa0bf9e8931ed03f7b2a/lib/SectionsManager.jsm#L33

Reply:
I would guess `values` comes from our previous usage of react-intl FormattedMessage https://github.com/formatjs/react-intl/blob/master/docs/Components.md#formattedmessage